programs/create-genesis/genesis_create.cpp

```diff
@@ -1003,33 +1003,6 @@ struct genesis_create::genesis_create_impl final {
         }
         // store messages
         const int n = _visitor.comments.size();     // messages count
-        db.start_section(config::system_account_name, N(gtransaction), "generated_transaction_object", n);
-        transaction tx{};
-        tx.actions.emplace_back(action{
-            {{_info.golos.names.posting, config::active_name}},
-            _info.golos.names.posting,
-            N(closemssg),
-            {}
-        });
-        const auto expiration_time = fc::hours(_info.golos.posts_trx.expiration_hours);
-        const auto delay = !_info.golos.posts_trx.initial_from ? microseconds{0} :
-            time_point_sec{_conf.initial_timestamp} - *_info.golos.posts_trx.initial_from;
```

Comment:
Please remove from `genesis-info` parameters for closemssg generated transactions (_info.golos.posts_trx)